nomad/job_endpoint_test.go

```diff
@@ -3988,6 +4016,16 @@ func TestJobEndpoint_ValidateJobUpdate(t *testing.T) {
 	} else {
 		t.Log(err)
 	}
+
+	new = mock.Job()
+	new.Dispatched = true
+	require.Error(validateJobUpdate(old, new),
```

Comment:
Generally you want to assert the error you are expecting. require.Contains(err.Error(), "foo bar")